fe/fe-core/src/main/java/org/apache/doris/nereids/glue/LogicalPlanAdapter.java

```diff
@@ -79,6 +81,10 @@ public ArrayList<String> getColLabels() {
         return colLabels;
     }
 
+    public List<View> getViews() {
```

Comment:
move views to StatementContext, u could get them from StatementContext directly

Reply:
fixed